### ISSUE TITLE
Accessibility issue with OverflowMenuItem component

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#5628]
## Title line template: [Title]: Brief description

Overflowmenu options are not accessible using keyboard.

-->

## Environment

> Operating system

Mac os 10.15.3

> Browser

Chrome

> Automated testing tool and ruleset

Dynamic Assesment Plugin
DAP rule set: IBM Accessibility January 2020

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
"@console/pal": "1.90.8"
 "carbon-components": "^10.10.0",
   "carbon-components-react": 7.10.1,

> What did you expect to happen?

We should be able to navigate oveflow menu items/options using keyboard 

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

Go to https://dev.console.test.cloud.ibm.com/security-advisor#/notifications
Add a notfication channel if not present
Open the menu by keyboard.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/46310374/76606187-d5a21f80-6537-11ea-8960-0bd5d43ba3dd.png)
![image](https://user-images.githubusercontent.com/46310374/76606195-db980080-6537-11ea-9e38-86a74a599384.png)

- Notes

**Comment by emyarod** [comment]
overflow menu should have arrow key navigation out of the box (https://github.com/carbon-design-system/carbon-components-react/pull/1822), 
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by reeteshsingh93** [comment]
We missed to use primaryFocus option in overflow menu items , hence it was not working for us. We are nit aware that for keboard accessibility it should be there.
